index.bs

```diff
@@ -256,8 +256,189 @@ exchanging control messages, results, and events.
 Presentation API Protocol {#presentation-api}
 ---------------------------------------------
 
-Issue(55): Incorporate material from the
-[Protocol](protocol.md) document.
+This section defines the use of the Open Screen Protocol for the
+[[PRESENTATION-API|Presentation API]].  When sections are quoted here, they are
+from the [[PRESENTATION-API|Presentation API]].  When the
+[[PRESENTATION-API|Presentation API]] refers to "implementation specific
+mechanisms", this spec defines the behavior when the choosen mechanism is the
+Open Sceen Protocol.  For all references messages, see Appendix A for the full
+CDDL definitions.
+
+When section 6.4.2 says "This list of presentation displays ... is populated
+based on an implementation specific discovery mechanism", the user agent, or
+controlling endpoint, may use the the mDNS, QUIC, and endpoint-info-request
```

Comment:
This is an interesting question.  This spec is trying to do two different things at the same time:

1. Define OSP, for use by any endpoints, including non-browser endpoints.

2. Define how browser should use OSP when implementing web APIs.

When doing #2, it does seem to make more sense to use the language of the APIs.  When doing #1, it might make more sense to use non-browser language.  When doing both at the same time, which language is better?  

One option might be to break it into two sections: one that's just #1 and one that's just #2, and then use different language in each section (as if we were writing two specs).

Reply:
+1 to the "two sections" option, as I suppose we want to define 1) in isolation, without referencing the Presentation API, with 2) being a mapping between API algorithm steps and relevant hooks in the protocol.

Splitting into different sections (or different specs in the end) might ease testing efforts as well, as I suppose we'll want to create a test suite at the API level, and a distinct test suite at the protocol level.

I don't know whether that split should be done right away, or if it's easier to proceed with one section for now, and refactor later on. Up to you :)

Reply:
I've split it into two sections.